### ISSUE TITLE
Start Request With okhttp3.Call

### ISSUE THREAD

**Issue by LegendaryCoders** [square/retrofit#2777]
Hello again

How can i start okhttp3.Call request with retrofit with out any interface class for request?

**Comment by JakeWharton** [comment]
Call `.enqueue()` or `.execute()` on an `OkHttpClient`. Why are you using Retrofit in that case?

**Comment by LegendaryCoders** [comment]
I want to convert okhttp3.Call to retrofit2.Call


Im try to wrap retrofit library for another ide so i don't know how i can create request for retrofit runtime.

**Comment by JakeWharton** [comment]
This isn't supported as it undermines most of the value of using Retrofit in the first place. Can you describe your use case more?

**Comment by LegendaryCoders** [comment]
`retrofit.callFactory.newCall(Request)`
This method will return okhttp3.call
Is there any way for convert okhttp3.call to retrofit2.call?

**Comment by JakeWharton** [comment]
No, there's nowhere to pull all of the information required at the Retrofit layer such as the converters in use.

**Comment by LegendaryCoders** [comment]
Retrofit is checking annotaions type . But i don't wany to use interface class methods
I want to start a custom request.
Retrofit has a class `RequestBuilder`
But i can't use it
